bin/bot.py

```diff
@@ -73,5 +82,22 @@ async def on_ready():
 async def on_guild_join(guild):
     await database.add_guild(guild)
 
+@bot.before_invoke
+async def command_logger(ctx):
+  # Logging any command run
+  server = ctx.guild.name
+  logging.info(str(server) + ":" + str(ctx.author) + " ran the command '" + str(ctx.message.content) +"'")
+
+""" Error Check """
+async def cog_command_error(ctx, error):

```

Comment:
`cog_command_error` will log any unhandled errors which pop up within a cog. So rather than it silently failing we can refer to the log file to see these errors.

`on_command_error` will log any errors when a user tries to call a cog. Eg. invalid name or too many paramaters supplied.